initfiles/componentfiles/configschema/xsd/types.xsd

```diff
@@ -47,20 +47,21 @@
     <xs:simpleType name="cronSchedule">
         <xs:restriction base="xs:string">
             <xs:pattern
-                    value="^(\*|((\*\/)?[1-5]?[0-9])) (\*|((\*\/)?[1-5]?[0-9])) (\*|((\*\/)?(1?[0-9]|2[0-3]))) (\*|((\*\/)?([1-9]|[12][0-9]|3[0-1]))) (\*|((\*\/)?([1-9]|1[0-2]))) (\*|((\*\/)?[0-6]))$"/>
+                    value="^(\*|([0-9]|1[0-9]|2[0-9]|3[0-9]|4[0-9]|5[0-9])|\*\/([0-9]|1[0-9]|2[0-9]|3[0-9]|4[0-9]|5[0-9])) (\*|([0-9]|1[0-9]|2[0-3])|\*\/([0-9]|1[0-9]|2[0-3])) (\*|([1-9]|1[0-9]|2[0-9]|3[0-1])|\*\/([1-9]|1[0-9]|2[0-9]|3[0-1])) (\*|([1-9]|1[0-2])|\*\/([1-9]|1[0-2])) (\*|([0-6])|\*\/([0-6]))$"/>
```

Comment:
May not be important, but it doesn't seem like the above regex would allow for lists or ranges in the cron schedule. Checked regex lib, there doesn't seem to be a simple expression that covers everything. This covers ranges and lists but is more than a little complicated: http://regexlib.com/REDetails.aspx?regexp_id=20201

Reply:
I did some searching for a regex to validate a cron schedule. I think some of the issues I had as I tested was that the jlib regex wasn't working, hence some expressions I tested may be been OK. I was looking for something that gave reasonable results. That one you mentioned, and by the way thanks for the reference to that site, is more complicated and probably beyond the scope of the simple schedules we want to validate.